virttest/utils_libvirt/libvirt_secret.py

```diff
@@ -3,12 +3,86 @@
 """
 
 import logging
+import re
+
+from aexpect import remote
+
+from avocado.core import exceptions
 
 from virttest import virsh
+from virttest.libvirt_xml import secret_xml
+from virttest.utils_test import libvirt
 
 LOG = logging.getLogger('avocado.' + __name__)
 
 
+def create_secret(params, remote_args=None):
+    """
+    Create a secret with 'virsh secret-define'
+
+    :param params: Test run params
+    :param remote_args: Parameters for remote host
+    :return: UUID of the secret
+    """
+    sec_usage_type = params.get("sec_usage", "volume")
```

Comment:
Rather than moving the place, I would suggest using setup_attrs() to update xml parts. setup_attrs() is our recommend way to setup xml setting. Please check "Libvirt automation tips in upstream" -> "Recommendation for library usage or specific checkpoints in avocado-vt" -> item 15.

Reply:
That is what is doing in the line 42 
```
 sec_xml.setup_attrs(**sec_dict)
```

Reply:
Thanks for your reminder ! But I think we'd better directly pass a dict with secret settings, instead of translating keys in params to new ones.

Reply:
I suggest we can move this PR forward in this way.
1)For current PR, we just focus on move this method into different file
2)Create separate one PR to create new method create_secret_xx() using so called pass a dict with secret settings, let user choose which one they want to use.

Reply:
Add deprecated log to create_secret func and not move function